scripts/assign.py

```
@@ -1,456 +1,27 @@
-#!/usr/bin/env python3
-import json
-import os
+import re
```

Ruff (0.8.2)

The file is executable but no shebang is present

(EXE002)